src/Nordea/Components/TextInput.elm

```diff
@@ -89,25 +75,36 @@ getAttributes config =
         , config.placeholder |> Maybe.map placeholder
         ]
 
-
-
 -- STYLES
-
-
-styles : List Style
-styles =
+getStyles : Config msg -> List Style
+getStyles config =
+    let
+        borderColorNormal =
+            borderColor Colors.grayMedium
+        borderColorStyle =
+            case config.error of
+                Just error ->
+                    if error then borderColor Colors.redDark
+                    else borderColorNormal
+
+                Nothing ->
+                    borderColorNormal
+
```

Comment:
```suggestion
    let
        borderColor =
            if config.error then
                Colors.redDark

            else
                Colors.grayMedium
```

Reply:
this gave me mismatch error since config.error is Maybe Bool, not Bool

Reply:
It fixed itself when removing Maybe from error 😊

Reply:
Yes should have mentioned that 😅